tests/Doctrine/Tests/Common/Cache/Psr6/DoctrineProviderTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Doctrine\Tests\Common\Cache\Psr6;
+
+use Doctrine\Common\Cache\CacheProvider;
+use Doctrine\Common\Cache\Psr6\DoctrineProvider;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
```

Comment:
you could use `CacheAdapter` instead here and get rid of `symfony/cache` in require-dev, isn't it?

Reply:
> The symfony/cache dev dependency is there to allow testing against an actual PSR-6 class - alternatively I could wrap a doctrine ArrayCache in a CacheAdapter and test against that, but it somehow seems wrong.

;)